gr-blocks/lib/message_debug_impl.cc

```diff
@@ -70,25 +105,8 @@ pmt::pmt_t message_debug_impl::get_message(int i)
     if ((size_t)i >= d_messages.size()) {
         throw std::runtime_error("message_debug: index for message out of bounds.");
     }
-
     return d_messages[i];
 }
 
-message_debug_impl::message_debug_impl()
```

Comment:
wait, this is just moved around. What are the "improvements" in this commit?

Can you split this commit into actual changes with a separate commit for moving things around?

Reply:
I split the move into a second commit (and i can remove it entirely if it is not desired).